### PR TITLE
Fix 'MemoryError' while executing corilla workflow step for large images

### PR DESCRIPTION
**Pull request by dvischi**

### CODE REVIEW

File: tmlib/workflow/corilla/stats.py [hunk 1]
```diff
@@ -87,8 +87,17 @@ def update(self, image, log_transform=True):
             logger.warn('skip image because it contains infinite values')
         else:
             self.n += 1
-            delta_mean = array - self._mean
-            self._mean = self._mean + delta_mean / self.n
+            delta_mean = (array - self._mean).astype(np.float32)
+            try:
+                # perform in-place operations for memory efficiency
+                norm_delta_mean = np.array(delta_mean, copy=True)
```
Comment (riccardomurri) [inline tmlib/workflow/corilla/stats.py]: I'd move this comment just before line 95 -- it's `self._mean` that is being updated in place, not `delta_mean` or `norm_delta_mean` which look like the target of the comment now.

Comment (dvischi) [reply tmlib/workflow/corilla/stats.py]: `norm_delta_mean /= self.n` is also an in-place operation which was the main reason for the refactoring due memory errors. If you'd like I could move the comment in front of the try-block.

Comment (riccardomurri) [reply tmlib/workflow/corilla/stats.py]: What I meant is that the comment is confusing because you say on line 92 that operations are in-place, and then on line 93 you take a *copy* of the `delta_mean` array...

It's fine if you move the comment out of the block, or reword it to state that `self._mean` is updated in-place, or move the comment down to line 95... Your call :-)

(Anyway, note that GitHub comments refer to the line *above* them, not *below* ;-))

**Comment by riccardomurri** [comment]
Merged, thanks!
